__tests__/app.test.js

```diff
@@ -116,6 +116,90 @@ describe('/api/articles/:article_id' , () => {
+              }
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchInfo)
+            .expect(200)
+            .then(({body: {article}}) => {
+                expect(article).toEqual(expected)
+            })
+        })
+        test('PATCH: 200 - returns the updated article when inc_votes is negative' , () => {
+            const patchInfo = {inc_votes: -5}
+            const expected = {
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 95,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              }
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchInfo)
+            .expect(200)
+            .then(({body: {article}}) => {
+                expect(article).toEqual(expected)
+            })
+        })
+        test('PATCH: 400 - Bad request returns an error when inc_votes is not a number' , () => {
+            const patchInfo = {inc_votes: 'string'}
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchInfo)
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toEqual('400 - Bad Request Invalid Data Type')
+            })
+        })
+        test('PATCH: 404 - Not found returns an error when ther article id does not exist' , () => {
+            const patchInfo = {inc_votes: 19}
+            return request(app)
+            .patch('/api/articles/100')
+            .send(patchInfo)
+            .expect(404)
+            .then(({body: {msg}}) => {
+                expect(msg).toEqual('404 - Article not found')
+            })
+        })
+        test('PATCH: 400 - Bad request returns an error when the patch request body does not contain the correct key', () => {
+            const patchInfo = {worng_key: 10}
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchInfo)
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toEqual('400 - Bad Request Incorrect Format')
+            })
+        })
+        test('PATCH: 400 - Bad request returns an error when the patch request body contains too many properties', () => {
+            const patchInfo = {worng_key: 10 , inc_votes: 7}
+            return request(app)
+            .patch('/api/articles/1')
+            .send(patchInfo)
+            .expect(400)
+            .then(({body: {msg}}) => {
+                expect(msg).toEqual('400 - Bad Request Incorrect Format')
+            })
+        })
+    })
```

Comment:
I'm not sure that this should be a 400 - realistically, wouldn't it ignore the keys it wouldn't accept? It still has the right information? 

It's up to you though! :D